Source/AzureSqlBulkFailover.ps1

```diff
@@ -7,20 +6,16 @@
 # Copyright 2023 Microsoft Corporation. All rights reserved. MIT License
 #Read input parameters subscriptionId and ResourceGroup
 param(
-    [Parameter(Mandatory=$false)]
-    [PSCustomObject]$ScriptProperties,

```

Comment:
I made changes to the script inputs to receive properties set in the manifest. Double-check merge